content/teknologi/altinnstudio/changelog/app-nuget/v4/whats-new.md

```diff
@@ -5,6 +5,17 @@ description: Overview of changes introduced in version 4
 toc: true
 ---
 
+## 4.10.2 (2021-07-15) - Text resources are loaded locally
+- The app will now load texts from the locally stored text resource files (config/texts/*) instead of retrieving them from Storage. Texts are still uploaded to Storage during deploy. The change is to remove unnecessary calls to Storage and to avoid an issue with caching that prevented new texts from being used immediately. [#6466](https://github.com/Altinn/altinn-studio/issues/6466), [#6415](https://github.com/Altinn/altinn-studio/issues/6415)
+- Fixed a bug where a filename with space in it could lead to a crash. [#6421](https://github.com/Altinn/altinn-studio/issues/6421)
+- The app will now provide security headers like X-Frame-Options, X-XSS-Protection, X-Content-Type-Options, and Referer-Policy.
```

Comment:
They have to add app.UseDefaultSecurityHeaders in Startup.Configure and the namespace Altinn.App.Api.Middleware for it to be used. New applications after altinn studio release v2021.29 will have it enabled by default.